apps/server/src/modules/rocketchat-user/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './rocketchat-user.module';
 export * from './service';
 export * from './domain';
+export * from './domain/testing/rocket-chat-user.factory';
```

Comment:
Please don't export this file directly - export it via `testing` dir barrel file and to make it even better, export the whole `testing` dir from the `domain`'s dir barrel file (`index.ts`) - this export will be not needed anymore then.

Reply:
done